README.md

```diff
@@ -20,15 +20,11 @@ In addition to exposing MapboxGL functionality to React apps, react-map-gl also
 
 ### Installation
 
-    npm install --save react-map-gl
+Using `react-map-gl` requires `node >= v4` and `react >= 15.4`.
 
-* `browserify` - react-map-gl is extensively tested with `browserify` and works without configuration.
```

Comment:
I recommend making the top level readme even shorter. Cut it to essentials and focus on referencing the main documentation. 

The reason is that duplicated information here has a high risk of getting out of date.